nf_core/lint/schema_lint.py

```diff
@@ -2,6 +2,22 @@
 
 import logging
 import nf_core.schema
+import jsonschema
+
+
+def remove_required_fields(schema):
+    """ Remove all required fields from a schema """
+    for group_key in schema["definitions"].keys():
+        group = schema["definitions"][group_key]
+        group_keys = list(group.keys())
+        if "required" in group_keys:
```

Comment:
Pretty sure that you can just do `if "required" in group:` 🤔

Reply:
that would be nice, thanks!